client/src/views/WorkspaceDetail.vue

```diff
@@ -268,10 +268,10 @@ export default {
           },
         }
         );
-        this.tableCreationErrors = [];
+        this.tableCreationErrors = null;
         this.update()
       } catch(err) {
-        this.tableCreationErrors.push(err.response.data.message);
+        this.tableCreationErrors = err.response.data.message;
```

Comment:
Is this the front-end bug? What was the buggy behavior?

Reply:
Yes. The bug here is I was adding to the array of errors that the v-text-field would display, assuming it would display them all. It for some reason was only displaying once. Also, I didn't really think it was necessary to add that complexity when in reality it would probably only display one error at a time, so I just made that a single message. I should probably rename that variable `tableCreationError` (singular), to clarify that.

Reply:
Gotcha, thanks for the explanation. I agree with you about changing it to the singular.